src/TutorialTemplate.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import meditation from './assets/image-meditation.jpg'
+
+
+export default function TutorialTemplate (){
+    return (
+    <TutorialWrapper>
+        <img src={meditation} alt=""/>
+        <h2>5 Minuten of...</h2>
+        <p>Lorem ipsum dolor sit amet consectetur Quos accusamus libero minus ullam, beataelibero minus ullam, beatae possimus nostr</p>
```

Comment:
Please try to avoid placeholder text. Write a first version of some real text that you can iterate on later.